apps/cowswap-frontend/src/legacy/state/swap/extension.ts

```diff
@@ -55,6 +56,8 @@ export function useTradeExactInWithFee({
 
   // external price output as Currency
   const outputAmount = stringToCurrency(quote.price.amount, outputCurrency)
+  const partnerFeeAmount = partnerFee ? outputAmount.multiply(bpsToPercent(partnerFee.bps)) : undefined
+  const outputAmountWithPartnerFee = partnerFeeAmount ? outputAmount.subtract(partnerFeeAmount) : outputAmount
```

Comment:
im not sure i understand the logic here. 

I'm confused with the with/without notation too.

so `outputAmountWithPartnerFee` deducts the partner fee from `outputAmount`? 
It was confusing to me why we had `outputAmountWithoutFee` but now that there's two fees, it feels strange these derived fields. 

What do we want to do exactly here?